app/views/foreman/unattended/snippets/_chef_client_omnibus_bootstrap.erb

```diff
@@ -0,0 +1,10 @@
+# install.sh will install latest chef client version
+curl -L https://www.chef.io/chef/install.sh | bash
+
+# if you want specific version you can use something like this instead of install.sh
+# yum install https://opscode-omnibus-packages.s3.amazonaws.com/el/6/x86_64/chef-11.10.0-1.el6.x86_64.rpm
```

Comment:
I must verify version compatibility (client 10 vs server 12 is my major concern) but yes, for most of distros we should be able to find packages.